Makefile

```diff
@@ -47,9 +47,14 @@ clean:
 
 update_submodules:
 	if [ ! -f $(FV3UTIL_DIR)/requirements.txt  ]; then \
-		git submodule update --init external/fv3gfs-util external/daint_venv; \
+		git submodule update --init external/fv3gfs-util ; \
 	fi
 
+update_submodules_venv: update_submodules
```

Comment:
Can we improve the naming here that `update_submodules` still does everything and have one that just does venv and one that just does utils if we need that functionality?

Reply:
The utils one is always needed, but the venv one is only needed when using a virtualenv. It has caused problems for users trying to use fv3core out of the box with docker, expanding a submodule that's not even used.  Almost everything in the Makefile is geared toward the container version of things. I could rename update_submodules to something like update_submodules_utils, but if there is another base submodule added, could instead call it update_submodules_base?